apps/iiif/annotations/models.py

```diff
@@ -155,6 +155,6 @@ def set_span_element(sender, instance, **kwargs):
         instance.svg = instance.oa_annotation['on'][0]['selector']['item']['value']
         instance.oa_annotation['annotatedBy'] = {'name': 'Me'}
         instance.content = instance.oa_annotation['resource'][0]['chars']
-        instance.resource_type = Annotation.COMMENTING
+        instance.resource_type = instance.oa_annotation['resource'][0]['@type']
```

Comment:
This gives us a resource type which Mirador has sent us, which is `dctypes:Text`.  While this is better than `oa:commenting`, it's still not `cnt:contentAsText`.  To complicate matters, the [relevant Web Annotation Data Model example](https://www.w3.org/TR/annotation-model/#embedded-textual-body) has standardized on `TextualBody`, as apparently the W3C standards committee [rejected Open Annotation's `contentAsText`](https://www.w3.org/TR/annotation-model/#changes-from-the-open-annotation-draft).

So this fix implements Mirador's functionality, which does not match Readux's sample data, neither of which match the WebAnnotations spec.  I suggest that we make sure this works in Readux's Mirador fork and in out-of-the-box Mirador, and call it "good" if it does.  For extra credit, we could translate both `cnt:contentAsText` and `dctypes:Text` to the new standard `TextualBody` if it works in both environments.  

cc @jayvarner @saracarl